deployment/docker-compose/README.md

```diff
@@ -0,0 +1,458 @@
+- Follow the installation procedure for installing Black Duck. Do not start the containers.  Skip that step.
+
+##### 8. Bring the containers up.
+- Copy the blackduck-alert.env file to the same location where the Black Duck docker-compose files are located.
+- Start the containers. 
+    ```
+    docker-compose -f <PATH_TO_BLACK_DUCK>/docker-compose/docker-compose.yml -f <PATH_TO_ALERT>/docker-compose/docker-compose.yml -f <PATH_TO_ALERT>/docker-compose/docker-compose.local-overrides.yml -p <PROFILE_NAME> up -d
+    ```
+- Be sure to replace <PROFILE_NAME> with the profile name you wish to use for your deployment i.e. blackduck.
+    ```
+    docker-compose -f <PATH_TO_BLACK_DUCK>/docker-compose/docker-compose.yml -f <PATH_TO_ALERT>/docker-compose/docker-compose.yml -f <PATH_TO_ALERT>/docker-compose/docker-compose.local-overrides.yml -p blackduck up -d
+    ```
+    
+    Note: Don't forget the -d option at the end of the command line to run the command as a daemon process otherwise the container logs will go to standard output and ```ctrl+c``` will stop the application.
+
+## Upgrading Alert
+You will bring down the profile and then re-deploy the profile.
+The steps in the upgrade procedure are the same as the installation procedure after bringing the profile down.
+
+### Standalone Upgrade
+1. Execute the command:  
+    ```
+    docker-compose -f ./docker-compose/docker-compose.yml -f ./docker-compose/docker-compose.local-overrides.yml -p <PROFILE_NAME> down
+    ```
+2. Follow the [Standalone Installation](#standalone-installation)
+
+### Upgrade with Black Duck
+1. Execute the command:
+    ```
+    docker-compose -f <PATH_TO_BLACK_DUCK>/docker-compose/docker-compose.yml -f <PATH_TO_ALERT>/docker-compose/docker-compose.yml -f <PATH_TO_ALERT>/docker-compose/docker-compose.local-overrides.yml -p <PROFILE_NAME> down
+    ```
+2. Follow [Installation with Black Duck](#installation-with-black-duck)
+
+## Certificates 
+This section describes how to configure the optional certificates.  Please verify beforehand if custom certificates or a certificate truststore must be used.
+
+### Using Custom Certificates 
+- Custom Certificates for the Alert web server to present to clients.
+
+    - Before you can use custom certificates for Alert you must have the signed certificate and key used to generate the certificate.
+
+        - WEBSERVER_CUSTOM_CERT_FILE - The file containing the customer's signed certificate.
+    
+            ```cp <PATH_TO_CERT_FILE> <PATH_TO_SECRETS>/WEBSERVER_CUSTOM_CERT_FILE```
+
+        - WEBSERVER_CUSTOM_KEY_FILE - The file containing the customer's key used to create the certificate.
+
+            ```cp <PATH_TO_KEY_FILE> <PATH_TO_SECRETS>/WEBSERVER_CUSTOM_KEY_FILE```
+            
+### Using Custom Certificate TrustStore
+- Custom java trust store file for the Alert server to communicate over SSL to external systems.
+
+    You must have a valid JKS trust store file that can be used as the Trust Store for Alert.  
```

Comment:
We should mention that this is the trust store that certificates will need to be imported into and the secret will need to be updated when Alert runs into certificate errors.